engine/classes/Elgg/Filesystem/Directory/Fly.php

```diff
@@ -85,7 +85,7 @@ public function getContents($path) {
 	 */
 	public function getFile($path) {
```

Comment:
throw docs? or is it already part of the inherited docs?

Reply:
added